samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/Application.java

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.sample.multilingual;
+
+import com.microsoft.bot.builder.ConversationState;
+import com.microsoft.bot.builder.Storage;
+import com.microsoft.bot.builder.UserState;
+import com.microsoft.bot.integration.AdapterWithErrorHandler;
+import com.microsoft.bot.integration.BotFrameworkHttpAdapter;
+import com.microsoft.bot.integration.Configuration;
+import com.microsoft.bot.integration.spring.BotController;
+import com.microsoft.bot.integration.spring.BotDependencyConfiguration;
+import com.microsoft.bot.sample.multilingual.translation.TranslationMiddleware;
```

Comment:
There seem to be a missing import:
`import com.microsoft.bot.sample.multilingual.translation.MicrosoftTranslator;`